### ISSUE TITLE
2.1.12 New preset didn't stay in its right position after restart

### ISSUE THREAD

**Issue by neiriod** [drawpile/Drawpile#821]
Try add a new preset A
![01](https://user-images.githubusercontent.com/49644585/68078691-588ecd00-fe16-11e9-9702-504d5c1c1284.PNG)

After restart drawpile
![02](https://user-images.githubusercontent.com/49644585/68078686-2b421f00-fe16-11e9-8c1f-42773d6eadc9.PNG)

Position of preset A is moved to head of the preset, which its right position should be in the end of the preset.

**Comment by callaa** [comment]
This is related to issue #820. Unfortunately, the fix is slightly involved, so it won't make it to today's release.

The problem basically is, that previously brush presets were saved in the configuration file or registry in an ordered list. In version 2.1.12, I changed it so the brushes are saved in individual files. (Arguably, the way it should have worked from the beginning.) When I did this, I had totally forgotten it was possible to reorder the brushes by drag&drop, so I didn't reimplement that part.
